### PR TITLE
Add doc and script for building to physical device

### PR DESCRIPTION
**Pull request by jonathangarelick**
Added documentation and an easy-to-use shell script so future contributors can build to their physical device.

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

### CODE REVIEW

File: prep_for_physical.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```
Comment (darinwf) [inline prep_for_physical.sh:1]: nit: Can you put this in the `Scripts/` folder instead?

File: Docs/BUILDING.md [hunk 2]
```diff
@@ -55,3 +55,11 @@ Run on the Simulator
 
 * Open `Client.xcodeproj` and make sure you have the *Client* scheme and a simulated device selected. The app should run on any simulator.
 * Select *Product -> Run* and the application should build and run on the selected simulator.
+
+Run on a Physical Device (requires an active Apple Developer subscription)
+-----------------
+
+* Open `Client.xcodeproj` and make sure you have the *Client* scheme and your registered device selected. You might need to give your device's UDID to a Neeva employee.
+* Execute our `prep_for_physical.sh` script.
```
Comment (darinwf) [inline Docs/BUILDING.md:63]: nit: Make this `Scripts/prep_for_physical.sh`

File: prep_for_physical.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (darinwf) [inline prep_for_physical.sh:2]: nit: Would be good to add a descriptive comment at the top of this file explaining what the script file is for. Letting people know that this is for use if you want to deploy the app to a device using your own custom Team ID, etc.

**Comment by jonathangarelick** [comment]
@darinwf Okay, I believe all your points should be addressed now. I also cleaned up the script a bit. Let me know if you need anything else, thanks!

**Review by darinwf — APPROVED** [review]

**Review by darinwf — APPROVED** [review]

**Pull request merged**